### PR TITLE
change vbuild to poi

### PR DESCRIPTION
**Pull request by kazupon**
Maybe, We need to update `play.config.js` generated by [getplay](https://github.com/vue-play/getplay) too.

### CODE REVIEW

File: config/config.play.js [hunk 1]
```diff
@@ -8,16 +9,13 @@ module.exports = {
...
   postcss: config.postcss,
-  templateCompiler: true,
-  webpack: {
-    resolve: {
-      modules: [
-        path.resolve('src')
-      ]
-    }
+  webpack (config) {
+    ['app', 'preview'].forEach(entry => {
```
Comment (kazupon) [inline config/config.play.js]: I hope you should add `hmrEntry` option to `poi` due to write path with hard coded.

Comment (egoist) [reply config/config.play.js]: Here's a special keyword `[hot]` for this:

```js
{
  entry: {
    app: ['[hot]', './play/app.js']
  }
}
```

But I think I will rename it to `:hot:` which looks less confusing. (`[hot]` will still work)

Comment (egoist) [reply config/config.play.js]: Just added this in [poi@8.0.4](https://github.com/egoist/poi/releases/tag/poi%408.0.4)

**Comment by egoist** [comment]
`getplay` has been updated too.

**Comment by kazupon** [comment]
👍🏼
Thanks

**Pull request merged**